src/components/partials/htmlHead.js

```diff
@@ -104,6 +106,15 @@ const HtmlHead = () => {
         href="https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Serif+4:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&display=swap"
         rel="stylesheet"
       />
+      {isProduction && (
+        <script
```

Comment:
The Gatsby `<Script />` component wasn't working. No errors, just nothing rendered.